TestAutomationFramework.BDT/Pages/HowToDoItPageSteps.cs

```diff
@@ -0,0 +1,32 @@
+﻿using NUnit.Framework;
+using TechTalk.SpecFlow;
+using TestAutomationFramework.Core.Browser;
+using TestAutomationFramework.TestData;
+using TestAutomationFramework.Web.PageObjects.Pages;
+
+namespace TestAutomationFramework.BDT.Pages
+{
+    [Binding]
+    public class HowToDoItPageSteps
+    {
+        HowWeDoItPage howWeDoItPage = new();
+        [When(@"I click the How we do it link on Header")]
+        public void WhenIClickTheHowWeDoItLinkOnHeader()
+        {
+            howWeDoItPage.Load();
+        }
+
+        [When(@"I click back")]
+        public void WhenIClickBack()
+        {
+            BrowserFactory.Browser.Refresh();
+            BrowserFactory.Browser.Back();
+            BrowserFactory.Browser.WaitForPageLoad();
+        }
+        [Then(@"I check the previous page")]
+        public void ThenICheckThePreviousPage()
+        {

```

Comment:
please remove this step and in feature replace with method that will check that how we do it page is opened